### DISCUSSION TITLE
New keys added in v5.0.0 related to distance between crystals

### DISCUSSION THREAD

**Discussion by yere-qubio** [UCL/STIR#1395] [Q&A] [answered]
Hi, i was wondering if the distances specified in the following keys (introduced in v5.0.0):
```
Scanner parameters:=
normal parameters...
Scanner geometry (BlocksOnCylindrical/Cylindrical/Generic)  := BlocksOnCylindrical
Distance between crystals in axial direction (cm)           := 0.22
Distance between crystals in transaxial direction (cm)      := 0.22
end scanner parameters:=
```
refer to distance between crystal edges (that means, the gap between crystals) or to distance between the center of the crystals (that means `gap + 2 * (axial crystal size / 2)`).

Thanks in advance.

**Comment by robbietuk** [comment, marked as answer]
> distance between the center of the crystals (that means gap + 2 * (axial crystal size / 2)).

This one. These variables refer to the pitch of the object. The same is used for the blocks.

**Reply by yere-qubio** [reply]
Thanks!
